app/models/katello/host/content_facet.rb

```diff
@@ -206,7 +206,7 @@ def calculate_and_import_applicability
         ::Katello::Applicability::ApplicableContentHelper.new(self, ::Katello::Erratum, bound_repos).calculate_and_import
         ::Katello::Applicability::ApplicableContentHelper.new(self, ::Katello::ModuleStream, bound_repos).calculate_and_import
         update_applicability_counts
-        self.update_errata_status
+        self.host&.refresh_statuses(::Katello::HostStatusManager::STATUSES)
```

Comment:
Personally I don't think this change is appropriate. Why should an errata status update affect all other statuses?

Reply:
If we don't do it here, where should we update the host's RHEL lifecycle status? And how can we ensure that it's updated regularly?

I thought about adding some sort of cron schedule or something, but this seemed much easier since we're already doing a regular status update of errata.

Reply:
How about Host status card? Could we call this method when the card is loaded?

Reply:
There are other places besides the host status card where we'd need updated information-- `hammer host status`, for one. :thinking:

Reply:
I thought the epic is for UI notification. 
We can add it for `hammer host status` if required.